packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -203,12 +205,7 @@ DatePicker.defaultProps = {
   startDate: undefined,
   endDate: undefined,
   windowWidth: undefined,
-  allowNativeDatepickerOnMobile: true,
-  validationState: 'default'
+  allowNativeDatepickerOnMobile: true
 };
 
-const UncontrolledDatePicker = uncontrollable(DatePicker, {
-  selectedDate: 'onChange'
-});
-
-export default withTheme(withWindowSize(UncontrolledDatePicker));
+export default withTheme(withWindowSize(DatePicker));
```

Comment:
This is mainly a reminder for myself to remove the `withTheme` usage.

@jrios I started looking at a `useWindowWidth` hook similar to [this gist and comments](https://gist.github.com/gaearon/cb5add26336003ed8c0004c4ba820eae). A lot of the tests around `DatePicker` use the `defaultWidth` prop to mock the state. Setting the screen size at the start of each of the affected tests didn't seem to resolve the errors I ran into. What are your thoughts on this?

Reply:
👍 on `useWindowWidth`. 

What errors are you referring to?

Reply:
I was running into errors around mocking `document.body.clientWidth` in the `DatePicker` tests. I've got it all working now in #295